corehq/ex-submodules/couchexport/tests/test_schema.py

```diff
@@ -110,7 +110,7 @@ def test_split_column_ignore_extras(self):
 
     def test_split_column_missing(self):
         self.post_it()
-        self._test_split_column([[None, None, None, None, None]])
+        self._test_split_column([[None, None, None, None, SCALAR_NEVER_WAS]])
```

Comment:
it looked like it was testing for one extra column that's not in the schema so it should just be one extra value of `SCALAR_NEVER_WAS`, is that not right? note that the `SCALAR_NEVER_WAS` will only be added for fields that should be included in the export but are not in the export schema.